Makefile

```diff
@@ -0,0 +1,5 @@
+.PHONY: build
+
+build:
+	rm -rf node_modules
+	docker run -it --entrypoint /bin/bash --workdir /indico-sui-theme --mount type=bind,src=`pwd`,target=/indico-sui-theme node:10 build.sh
```

Comment:
run the container as your own uid, no need to build as root (and more important, no need to `npm install` all the stuff as root). also, can we avoid creating the `node_modules` folder outside the container?

Reply:
I could probably mount a tmpfs volume at the container node_modules to prevent modification of the host directory